package.json

```diff
@@ -98,7 +98,7 @@
     "packages/*"
   ],
   "engines": {
-    "node": "16.13.2"
+    "node": "16.13"
```

Comment:
don't you think because you and I and some other collaborator might already have another patch version of node 16.13 installed on their end. less anoying and between patches of node nothing should break ?.

If that is really annoying for you I can revert

Reply:
Not annoying is just that I think we want to be using the same version of NodeJS as PKG. At the same time I agree that it should be okay to discard the path version here!